### ISSUE TITLE
[Suggestion] Customized ore cluster

### ISSUE THREAD

**Issue by wormzjl** [pWn3d1337/Techguns#59]
Well if possible
So one can create their own special ore cluster block

**Comment by pWn3d1337** [comment]
The problem with ore clusters is that it requires block ids. Each type requires an own metadata value.

**Comment by wormzjl** [comment]
might consider to make it tile-entity based?
you can decrease the block count in the meantime

**Comment by pWn3d1337** [comment]
No, that's bad for performance. I don't want tile entities generating in the world.

**Comment by wormzjl** [comment]
nervermind

**Comment by wormzjl** [comment]
Ok got a new idea, why not just "reserve" a few block IDs and let people fill them?
Seems a bit dirty but actually works...

**Comment by pWn3d1337** [comment]
This is actually planned for the next release, I'm currently adding minetweaker support and I think a good idea wood be a new OreCluster block with 16 open metas that can be defined with minetweaker.

**Comment by wormzjl** [comment]
This is great :)

**Comment by pWn3d1337** [comment]
The newest version 1.2 alphatest3 has support for custom ore clusters

**Comment by wormzjl** [comment]
Wow thank you for the quick update

**Comment by wormzjl** [comment]
![image](https://cloud.githubusercontent.com/assets/2989009/25307353/191637c0-27a0-11e7-8a60-f17f8df101e4.png)
It's working. Incredible job. I didn't thought this can be done only by minetweaker.

However in the wiki page the item example should come with <>
OreCluster.addOreCluster(0,<minecraft:planks>,1,0.4,0.4,0.0,3,0.5,2.5);

In my testing it seems that I cannot use liquid as a output, my script is 
mods.techguns.OreCluster.addOreCluster(1,<liquid:water>,1,0,0,1,3,0.9,1.5);
Is liquid currently unsupported or it's my grammar issue?

In the end, as I've stated previously, would you mind adding another customizable block?

Thank you again for the addition of this feature

**Comment by pWn3d1337** [comment]
Liquid is not implemented for minetweaker yet, but I could add that. A second custom block? Are 16 types not enough? Yes I could add a second one.

**Comment by wormzjl** [comment]
![image](https://cloud.githubusercontent.com/assets/2989009/25307697/a232e0da-27a5-11e7-9b99-ea800ab4fe36.png)
![image](https://cloud.githubusercontent.com/assets/2989009/25307701/a6f8cdfa-27a5-11e7-8c08-ec24c379a09c.png)
Well, 32 is merely enough for TFC
